os/include/tinyara/pm/pm.h

```diff
@@ -199,12 +186,19 @@ enum pm_state_e {
 
 /* This structure is used to send data to pm driver from app side for timedSuspend */
 struct pm_suspend_arg_s {
-	enum pm_domain_e domain;          /* domain to be suspended */
+	int domain_id;                    /* domain ID to be suspended */
```

Comment:
Do weed no support negative numbers for the domain id?

Reply:
The domain_id can be seen as similar to the fd. 
Negative numbers mean errors.